### PR TITLE
Fix(eos_designs): management interface for 750 platforms

### LINKED ISSUE #3557: incorrect management interface for 750 platform
### Issue Summary

CCS-750 platform uses the logical Management0 to represent the current active management interface, but AVD tries to configure Management1, which is invalid on this platform

### Which component(s) of AVD impacted

eos_designs

### How do you run AVD ?

Ansible CLI (with virtual-env or native python)

### Steps to reproduce

_No response_

### Relevant log output

_No response_

### Contributing Guide

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by matthewgottlieb**
## Change Summary

The default eos_designs platform settings group the modular 750 platform (including specific names for its 5-slot and 8-slot variants) in with fixed platforms, which have different names for their management interfaces.  This change corrects the default management interface for the 750 platform to Management0.

## Related Issue(s)

Fixes #3557 

## Component(s) name

`arista.avd.eos_designs`

## Proposed changes
Divide the existing default platform setting list item into two, creating a new one for the 750 (and 755 and 758). 
 This new list item has the management interface set to Management0, with all other attributes copied.

### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

**Comment by ClausHolbechArista** [comment]
I believe these default platform_settings are hardcoded in the docs somewhere. Please update accordingly (and correct if you find other differences). Thanks!

File: ansible_collections/arista/avd/roles/eos_designs/docs/tables/platform-settings.md [hunk 1]
```diff
@@ -120,10 +120,18 @@
         - feature_support:
             poe: true
             queue_monitor_length_notify: false
+          management_interface: Management0
           platforms:
           - '750'
           - '755'
           - '758'
+          reload_delay:
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_designs/docs/tables/platform-settings.md:128]: Sorry for pointing you in this direction. The default value is defined in `ansible_collections/arista/avd/roles/eos_designs/schema/schema_fragments/platform_settings.schema.yml`. After updating the fragment you need to run `pre-commit run schemas --all` to regenerate schemas and documentation.

Comment (matthewgottlieb) [reply ansible_collections/arista/avd/roles/eos_designs/docs/tables/platform-settings.md:128]: No worries—thank you for the guidance!

File: ansible_collections/arista/avd/tests/integration/requirements.txt
Comment (matthewgottlieb) [inline ansible_collections/arista/avd/tests/integration/requirements.txt:1]: Not sure what happened here, probably best for me to start over at this point :)

**Comment by ClausHolbechArista** [comment]
Moving to draft until rebasing is done. Seems like something went wrong in the merge/rebase regarding symlinks. The requirements files are symlinked a few places and here the links are being replaced with regular files.

**Review by ClausHolbechArista — APPROVED** [review]

**Review by carlbuchmann — DISMISSED** [review]
LGTM

**Review by gmuloc — APPROVED** [review]
LGTM thanks!

**Pull request merged**